cspatients/util.py

```diff
@@ -56,10 +66,13 @@ def get_all_active_patient_entries(search=None, status=None):
                 urgency=status, completion_time__isnull=True
             )
 
+    # latest most urgent on the top, completed at the bottom
     sorted_entries = sorted(
-        patiententrys, key=lambda x: (x.urgency, x.decision_time), reverse=True
+        patiententrys, key=lambda x: (x.decision_time), reverse=True
+    )
+    return sorted(
+        sorted_entries, key=lambda x: (x.completion_time is not None, x.urgency)
```

Comment:
I'm just wondering whether we want to split this out into a separate, testable function so that we can check that the ordering is working as expected.
Alternatively, could we combine the `sorted` function into a single lambda?
Not super important

Reply:
I need to call `sorted` separately, only the first one is `reverse=True`.